java/test/jmri/jmrit/beantable/LogixTableActionTest.java

```diff
@@ -188,12 +188,14 @@ public void testDeleteLogix() {
     }
 
     void createModalDialogOperatorThread(String dialogTitle, String buttonText) {
-        new Thread(() -> {
+        Thread t = new Thread(() -> {
```

Comment:
You are aware that anonymous threads can be named and started in a single statement using the constructor
```
new Thread(() -> {
    ...do something interesting...
}, "your name here").start();

Reply:
That's fine, but this is a little more clear.  Plus, we may want to join the threads late, so we need to retain access to them.